### ISSUE TITLE
Pennsylvania

### ISSUE THREAD

**Issue by kcoronel** [hackforla/ballotnav-states#46]
### Overview
The database or data has some formatting or other issues that need to be resolved before it can be imported.

### Action Items
- [x] Data Collection Design
   - [x] Review Secretary of State Website
   - [x] Call a County starting from the first until you reach one that can validate the data with you.
      - [x] Checking the data that we have is correct
      - [x] Gathering data for fields that are empty
      - [x] Asking if there is anything else we should know
   - [x] Update Location Validation Test Sheet
   - [x] Edit the issue Action Items to reflect findings
- [x] Data Scientist/Scrapers
   - [x] Add the following columns to your data schema
      - [x] display_notes
      - [x] where
      - [x] start_date
      - [x] accessibility
   - [x] Delete the following fields
      - [x] field 23 
      - [x] latitude 
      - [x] longitude 
      - [x] social
      - [x] state and counties and county reps copy
   - [x] Rename the following fields
      - [x] validate_url to source_data
   - [x] If Hours are available add them to the sheet by using the following convention (removing any info about days they are closed e.g. closed sat and sun, unless it refers to a specific date they are closed):
MONDAY-FRIDAY:1:00 PM-4:30PM  or Monday-Friday:7:00-12:00
- [x] Address fields needs to be broken up into individual components (address_1 and address_2) 
   - address_1 = Street name and address #
   - address_2 = suite # or room #, etc
   - city = city name
   - state = state name spelled out
   - state_2 = two-letter abbreviation for the address (for some states the location is in a different state than the voting district (crazy I know.  But this is why there is a state field separate which is supposed to be the state spelled out and that is the jurisdiction, not the physical location)

### Resources
[Google Sheet](https://docs.google.com/spreadsheets/d/1jAuDt7vJZNMC-91maU-6x9UO0e-Rv1ESiylnEbPMMzA/edit?usp=sharing)

**Comment by kcoronel** [comment]
**Drop Boxes and Dropoff Sites**
Drop boxes are a safe and secure way for Pennsylvania voters to return their voted mail-in or absentee ballots. Voters may return their mail-in or absentee ballot at their county election board's designated drop box(es) or dropoff location before 8 p.m. on election day, Tuesday, November 3, 2020.

Services available at a drop box or dropoff location:
Return only your own voted ballot
NOTE: Voted ballots may also be returned at your county election office or satellite election office. The list of drop box locations will be updated as more information becomes available.

**Satellite County Election Offices**
Some county election boards have opened satellite offices to offer county residents added convenience. These satellite locations may be open additional hours, including weeknights and weekends. Each satellite location has a secure ballot receptacle to store voted mail-in and absentee ballots submitted at the location. 

Services available at a County Election Office or Satellite County Election Office:

Register to vote
Vote early in person
Apply for a mail-in or absentee ballot
Vote your ballot, once your application is verified
Return a completed ballot
Return a completed ballot that was mailed to you.
The list of satellite locations designated by the county election board will be updated as information is made available.

**Comment by kcoronel** [comment]
@Arjayellis @ExperimentsInHonesty this is an updated list with additional [drop off locations](https://www.votespa.com/voting-in-pa/pages/drop-box.aspx)

**Comment by kcoronel** [comment]
@Seymour100 found a new list of additional [drop off locations](https://www.votespa.com/voting-in-pa/pages/drop-box.aspx) for PA that was recently posted. Do you have capacity to get it re-scrapped to include additional locations

**Comment by Seymour100** [comment]
@kcoronel yes, I think I can do this one all in Google Docs since the information is presented in a table.

**Comment by kcoronel** [comment]
awesome @Seymour100 keep us posted!

**Comment by Seymour100** [comment]
@kcoronel , I added a second sheet for this data so as not to disturb the first scrape. I also added a column "day" since some hours applied to specific days. Should be ready now.

**Comment by kcoronel** [comment]
I combined all of the listed locations into sheet 2-it is the complete list for PA as mentioned [here](https://www.votespa.com/voting-in-pa/pages/drop-box.aspx)

**Comment by aNullValue** [comment]
Pennsylvania has been processed.
